.circleci/config.yml

```
@@ -18,7 +18,7 @@ jobs:
             rm -rf build
             mkdir build
             cd build
-            cmake -DCMAKE_BUILD_TYPE=Debug -DIEX_ENABLE_ALLWARNINGS:BOOL=ON -DIEX_BUILD_TESTING:BOOL=ON ..
+            cmake -DCMAKE_BUILD_TYPE=Debug -DIEX_BUILD_DOCUMENTATION:BOOL=OFF -DIEX_ENABLE_ALLWARNINGS:BOOL=ON -DIEX_BUILD_TESTING:BOOL=ON ..
```

CI should probably build doc target to make sure it works.
```suggestion
            cmake -DCMAKE_BUILD_TYPE=Debug -DIEX_BUILD_DOCUMENTATION:BOOL=ON -DIEX_ENABLE_ALLWARNINGS:BOOL=ON -DIEX_BUILD_TESTING:BOOL=ON ..
```